crates/datasources/src/excel/table.rs

```diff
@@ -22,7 +25,16 @@ pub async fn excel_streaming_table(
 ) -> Result<Arc<dyn TableProvider>, ExcelError> {
     let mut workbook: Xlsx<_> = match source_url {
         DatasourceUrl::File(path) => open_workbook(path)?,
-        DatasourceUrl::Url(_) => todo!("remote files not supported yet"),
+        DatasourceUrl::Url(_) => {
+            /*
+            let store = store_access.create_store().unwrap();
+            let buffer = object_store::buffered::BufReader::with_capacity(store, &meta, infer_schema_length);
+            let sheets = calamine::open_workbook_auto_from_rs(buffer).unwrap();
+             */
```

Comment:
is this temporary or should we delete it?

Reply:
temporary, was an experiment that I intended to come back to, deleting.